travis/setup_dependencies_common.sh

```diff
@@ -38,9 +38,12 @@ fi
 if [[ $NUMPY_VERSION == dev ]] || [[ $NUMPY_VERSION == development ]]; then
     $PIP_INSTALL git+http://github.com/numpy/numpy.git
     export CONDA_INSTALL="conda install python=$PYTHON_VERSION"
-else
-    conda install  numpy=$NUMPY_VERSION
+elif [[ ! -z $NUMPY_VERSION ]]; then
+    conda install numpy=$NUMPY_VERSION
     export CONDA_INSTALL="conda install python=$PYTHON_VERSION numpy=$NUMPY_VERSION"
+else
+    conda install numpy
+    export CONDA_INSTALL="conda install python=$PYTHON_VERSION numpy"
```

Comment:
Since there is no version restriction you don't need to list numpy again. Or alternatively if a package doesn't give a numpy version there's no need to install it in the line before, just include it in ``$CONDA_INSTALL`` and deal with it with the rest of the dependencies?